### ISSUE TITLE
Any way to unlock spells?

### ISSUE THREAD

**Issue by Nyxistaken** [Sorrow446/HLSE-Hogwarts-Legacy-Save-Editor#7]
Hey! Do you think it would be possible to unlock spells through the save files?

I am on a 2nd character and don't want to go through the tedious process of unlocking all the spells again

**Comment by Sorrow446** [comment]
Yeah, I'd need to do some testing. Think it's these: STAT_Lumos_Played, STAT_Stupefy_Played
Your completed save file could help if you've still got it please.

**Comment by Nyxistaken** [comment]
Here you go, this should contain my first char on save one and the new char
on save two
https://www.mediafire.com/folder/vpby4ygnhn6kb

I can see the line you mentioned, I am opening it with Notepad++ but the
value shows up a few times so not sure what to edit

**Comment by Sorrow446** [comment]
Thanks, I'll keep you updated.

**Comment by Devidkee** [comment]
Hello! I have found a way to unlock spells.
Create new records in the LocksDynamic table with the LockID set to the name of the spell (example: Spell_Crucio) and the ELockState set to 0 (it is always 0).

Here is the list of all spells (except Nab-Sack, Beast Feed and Beast Petting Brush):
Spell_Accio
Spell_Alohomora
Spell_Alohomora_L2
Spell_Alohomora_L3
Spell_ArrestoMomentum
Spell_Avadakedavra
Spell_Confringo
Spell_Conjuration
Spell_Crucio
Spell_Depulso
Spell_Descendo
Spell_Diffindo
Spell_Disillusionment
Spell_Expelliarmus
Spell_Expulso (Bombarda)
Spell_Finishers (Ancient Magic)
Spell_Flipendo
Spell_Glacius
Spell_Imperius
Spell_Incendio
Spell_Levioso
Spell_Lumos
Spell_Oppugno (Ancient Magic Throw)
Spell_Petrificus
Spell_Protego
Spell_Reparo
Spell_Revelio
Spell_Stupefy
Spell_Transformation (Altering spell)
Spell_TransformationOverland (Transformation spell)
Spell_Vanishment
Spell_Wingardium

Edit: the list now includes all spells (except vivarium spells)

**Comment by Nyxistaken** [comment]
Awesome, thanks!

**Comment by altermere** [comment]
Can't seem to unlock Transformation and Bombarda. Anyone had any luck?

**Comment by Devidkee** [comment]
> Can't seem to unlock Transformation and Bombarda. Anyone had any luck?

I am not near my computer at the moment so I may be wrong but I think Bombarda is ~~Spell_Oppugno~~ Spell_Expulso and for transformation you need to add Spell_TransformationOverland

When I arrive home I will write a complete list of all spells.

**Comment by Devidkee** [comment]
> Can't seem to unlock Transformation and Bombarda. Anyone had any luck?

Transformation is Spell_TransformationOverland
Bombarda is Spell_Expulso

I updated the list of spells. See my comment above.
